src/app/containers/CpsRelatedContent/index.jsx

```diff
@@ -32,6 +36,16 @@ const StyledSectionLabel = styled(SectionLabel)`
   margin-top: 0;
 `;
 
+// Apply the right margin-top between the section label and the promos
+const SpacingDiv = styled.div`
```

Comment:
would it be worth renaming this to `SingleContentWrapper` or something else with less ambiguity ?